### DISCUSSION TITLE
keri-dev 05/18/23

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#26] [Meetings]
[Recording](https://us06web.zoom.us/rec/share/zTgC0ciV38lWs4_3Tq8kWYq2fI9cBvYqof9utQMHOuDVtJcK3SLSJ6BgMgW0OXnA.Bls7pUrKRzJdqs0v) Passcode: VjiRCX.3

Contributions:
GLEIF (Kevin): Signifide/Keride/CESRide
GLEIF (Phil): KERIA/Signifypy - Delegation between KERIA/Signifypy
RootsID (on behalf of) Provenant (Alex/Rodolfo): Signify-TS - sample react app, authorization headers
SHIMs - Trezor/HSMs (PRs pushed)
Kent looking at a how to consume signify from mobile applications.
Henk, King of documentation! 👑 
Ubisecure (Petteri): Signify-TS - signature headers should be closer to the spec (looking issue/pr)
FFI is missing from CESRide (intentionally) but could be used for a general approach to generate the signature headers (looking for PR)
QUI (Jason): Signifide/Keride/CESRide/Parside 
Provenant (Arshdeep): Signify/KERIA

Interop:
CESR interop between CESRide/KERIpy/Signify-TS (issues needed)
Signify interop between signifypy/signifide/signify-ts (issues needed)
KERI interop (issues needed)

Created:
https://github.com/WebOfTrust/signify for the signify protocol.

**Comment by SmithSamuelM** [comment]
We should add the zoom link and meeting info to the keri readme.  Update the info to replace the keri concepts meeting now mentioned there.

Maybe add the zoom link to the top of this discussion
